src/Compilers/CSharp/Test/Semantic/Semantics/StaticNullChecking.cs

```diff
@@ -14572,7 +14584,7 @@ class C : I
 {
     void I.M<T>(T? x) { }
 }";
-            var compilation = CreateStandardCompilation(new[] { source, attributesDefinitions }, parseOptions: TestOptions.Regular.WithNullCheckingFeature());
+            var compilation = CreateStandardCompilation(new[] { source, attributesDefinitions }, parseOptions: TestOptions.Regular.WithNullCheckingFeature(includeNonNullChecking: false));
```

Comment:
I assume that you added `includeNonNullChecking: false` to the tests that failed without it. 
Would it be worthwhile to add it to some other tests that don't need it, or maybe duplicate them? They should still behave the same.